task_10/style.css

```diff
@@ -0,0 +1,385 @@
+    margin: 20px 0px;
+    max-width: 1920px;
+}
+.orange_text span {
+    color: orange;
+    font-weight: bold;
+}
+.rabbit_text {
+    width: 250px;
+    font-size: 15px;
+    text-align: justify;
+    margin: 20px 0px;
+    max-width: 1920px;
+}
+.nibs_flex{
+    display: flex; 
+    flex-direction: column; 
+    align-items: center;
+}
+.hoof_flex {
+    flex-grow: 0.4;
+}
+.label {
+    display: flex;
+    max-width: 1920px;
+}
+.input_flex {
+    flex-grow: 0.7;
+}
+.name {
+    color: orange;
+    font-size: 30px;
+    width: 10%;
+    padding-left: 100px;
+}
+.input {
+    border: inset;
+    height: 25px;
+    margin: 10px;
+    flex-grow: 0.9;
+}
+.input_message {
+    height: 100px;
+}
+.submit {
+    width: 150px;
+    height: 50px;
+    font-size: 30px;
+    background-color: orange;
+    color: white;
+    line-height: 50px;
+    text-align: center;
+    box-shadow: 0 -3px rgb(206, 166, 34) inset;
+    cursor: pointer;
+    margin-bottom: 20px;
+    max-width: 1920px;
+}
+.submit_main_flex {
+    display: flex; 
+    justify-content: flex-end; 
+    margin-right: 100px;
+}
+
+.footer {
+    font-family: STMarie;
+    background-color: orange;
+    color: white;
+    font-size: 30px;
+    text-align: center;
+    text-transform: uppercase;
+    height: 50px;
+    line-height: 50px;
+}
+.menu {
+    display: none;
+}
+.menu_word{
+    text-decoration: none;
+    color: gray;
+    font-weight: bold;
+    cursor: pointer;
+    padding-right: 100px;
+    font-size: 30px;
+}
+.menu_texst  {
+    color: gray;
+    font-weight: bold;
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+}
+#okno {
+    display: none;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    background-color: orange;
+    height: 155px;
+    }
+#okno:target {
+    display: block;
+}
+.close {
+    display: inline-block;
+    border: 1px solid grey;
+    font-size: 30px; 
+    color: black;
+    padding: 0 12px;
+    text-decoration: none;
+    background: #f2f2f2;
+    cursor:pointer;
+    }
+@media (max-width: 1024px){
+.name{
+    width: 20%;
+    }
+}
+@media (max-width: 900px){
+.lorem_flex{
+    margin-right: 0px;
+    flex-wrap: wrap;
+    }
+}
+@media (max-width: 900px){
+.rabbit_flex{
+    flex-wrap: wrap;
+    }
+}
+
+@media (max-width: 800px){
+.footer{
+    height: 100%;
+    }
+}
+
+@media (max-width: 800px){
+.sprite{
+    display: none;
+    }
+}
+@media (max-width: 800px){
+.menu{
+    display: block;
+    text-decoration: none;
+    font-size: 30px;
+    color: orange;
+    }
+}
+@media (max-width: 800px){
+.header_section{
+    padding-left: 100px;
+    padding-right: 0px;
+    }
+}
+@media (max-width: 600px){
+.big_orange_word_flex{
+    width: 300px;
+    }
+}
+@media (max-width: 600px){
+.like{
+    width: 200px;
+    font-size: 15px;
+    text-align: center;
+    }
+}
+@media (max-width: 465px){
+.header_section{
+    display: none;
+    }
+}
+@media (max-width: 465px){
```

Comment:
Why do you use the same @media(max-width: 465px)?
You can set all this styles in one
@media(max-width: 465px) {
.big_orange{
    margin-top: 54px;
    }
.menu_texst{
    margin-top: 20px;
    font-size: 15px;
    }
}

Reply:
Done.